secure_message/resources/threads.py

```diff
@@ -20,14 +22,73 @@ def get(thread_id):
         logger.info("Getting messages from thread", thread_id=thread_id, user_uuid=g.user.user_uuid)
 
         conversation = Retriever().retrieve_thread(thread_id, g.user)
+        conversation_metadata = Retriever.retrieve_conversation_metadata(thread_id)
 
         logger.info("Successfully retrieved messages from thread", thread_id=thread_id, user_uuid=g.user.user_uuid)
         messages = []
         for message in conversation.all():
             msg = message.serialize(g.user, body_summary=False)
             messages.append(msg)
+
+        if conversation_metadata.is_closed:
+            return jsonify({"messages": add_users_and_business_details(messages),
+                            "is_closed": conversation_metadata.is_closed,
+                            "closed_by": conversation_metadata.closed_by,
+                            "closed_by_uuid": conversation_metadata.closed_by_uuid,
+                            "closed_at": conversation_metadata.closed_at})
+
         return jsonify({"messages": add_users_and_business_details(messages)})
```

Comment:
We should consider returning `is_closed: false`, as this could simplify a clients use of the api - `is_closed` would always be present removing the need to check for it's existence before getting the value.